src/lib/mongoBackend/mongoRegistrationCreate.cpp

```diff
@@ -0,0 +1,204 @@
+/*
```

Comment:
Again, we will end with mongoRegistrationCreate and mongoCreateSubscription at the same time. Crazy :)

If style mandates the former, we later should be changed as soon as we end with registrations stuff.

NTC

Reply:
Yes, style mandates the former I'm afraid ...